### PR TITLE
Origin/meet the team page

### PR DESCRIPTION
**Pull request by ishipandey20**

### CODE REVIEW

**Comment by maryellenmarino** [comment]
The page is showing up blank on my end. Any idea what's going on? Has this been tested before creating a PR for it?

File: frontend/src/Pages/MeetTheTeamPage/MeetTheTeamPage.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import Row from 'react-bootstrap/Row';
+import Tab from 'react-bootstrap/Tab';
+import Button from 'react-bootstrap/Button';
+
+const MeetTheTeamPage = () => {
+    const teamMembers = [
+      {
+        name: 'Maryellen Marino',
+        role: 'President',
+        avatar: '',
```
Comment (maryellenmarino) [inline frontend/src/Pages/MeetTheTeamPage/MeetTheTeamPage.jsx:13]: Sending you both everyone's avatar now. I'll also send over Jr Directors shortly and after that we can also add Project Leads and potentially Organizers.

**Comment by maryellenmarino** [comment]
This looks like a great start! Here are some comments...

## Needs Fixing
- Looks like we're missing John as of right now so that should be added in before first merge
- When adjusting to mobile screen size, the title goes away
- At an odd tablet sizing, the boxes all become different odd shaped (though this isn't a huge deal for right now)

## Notes
- I will soon be sending over images of both directors and jr directors (along with their titles), we haven't decided if we want to use real images or not but that should be an easy and quick replace if so

## Needs to be ready for merge
- adding john and fixing the missing mobile title should be ready to go before we merge in

Great work everyone!!

**Comment by ths22115** [comment]
THIS LOOKS AWESOME! **Super** well done so far guys! I am a little confused as to where the mentor and volunteer buttons are supposed to be on the page lol (I see the code for them but they don't render) but besides that this is amazing.

File: frontend/src/Pages/MeetTheTeamPage/Mentors-Volunteers-buttons.jsx
Comment (ths22115) [inline frontend/src/Pages/MeetTheTeamPage/Mentors-Volunteers-buttons.jsx:1]: Following up on my last comment, maybe consider:
- moving this file to 'Components'
- creating a component for the buttons and blurb
- then just import the component to 'MeetTheTeamPage.jsx' and call it in the return for the page

I'm not completely sure if we want the buttons there considering all the rerouting we're doing but if you guys haven't planned out anything for them yet and/or Mel hasn't given you any direction for them yet, I'd just say go with bullet points above. Worst case is we just call the component on some page other than 'MeetTheTeamPage.jsx'.

**Pull request closed without merging**